### ISSUE TITLE
Jitsi Screen Sharing Extension Failed To Install

### ISSUE THREAD

**Issue by redyetidave** [jitsi/jitsi-meet#3716]
I'm not on a custom domain, I'm using https://meet.jit.si/ and the screensharing extension gives me the error;

Screen sharing extension failed to install

![jitsi_meet](https://user-images.githubusercontent.com/6065890/50229027-a3902500-03a9-11e9-8506-a32cd83d01d9.png)

I have googled around and can't see how I might fix this.

Dave.

**Comment by damencho** [comment]
This is known, and should already be fixed upstream, but not on meet.jit.si.

@saghul this is what I meant today, chrome is currently removed the inline install API and getDisplayMedia is not turned on yet. We need another meet.jit.si release to update this :(

**Comment by saghul** [comment]
Dammit.

**Comment by damencho** [comment]
Yep, I missed it and didn't knew there will be such gap. But @aaronkvanmeerten had in mind an easy fix for meet.jit.si, so we may fix it later today :) so stay tuned.
